extra/accel/ane/lib/ane.py

```diff
@@ -8,13 +8,13 @@
 import struct
 faulthandler.enable()
 
-basedir = os.path.dirname(os.path.abspath(os.path.realpath(__file__)))
+basedir = Path(__file__).resolve().parent
```

Comment:
Absolute path of a canonical path will be the same always, so it suffices to stick to canonical path through `resolve()`